doc/developer/debugging.rst

```diff
@@ -79,8 +82,26 @@ To obtain additional debug messages, you can rebuild your project as follows:
 
 Actually we display the running rule and why she is running (dependence update).
```

Comment:
I know this is not part of this PR, but I don't understand this sentence.

Reply:
I change it to:
```
It will add a message at the start of each rule with the list of files that required an update.

.. code::

   Build /build/c2ctemplate-cache.json due modification on vars.yaml
   -rw-r--r-- 1 sbrunner geomapfish 1321 2019-01-09 16:59:20.845623078 +0000 /build/c2ctemplate-cache.json
   -rw-rw-r-- 1 sbrunner geomapfish 1299 2019-01-10 08:31:35.707376156 +0000 vars.yaml
```
is it clearer?

Reply:
Yes, thanks